legwork/utils.py

```diff
@@ -377,3 +380,202 @@ def ensure_array(*args):
         else:
             array_args[i] = args[i]
     return array_args, any_not_arrays
+
+def get_theta_phi(ra, dec):
+    """Calculates the spherical inclination from the pole coordinate
+    using astropy coordinates
+    Parameters
+    ----------
+    ra : `float/array`
+        right ascension of the source. Must have astropy angular units
+
+    dec : `float/array`
+        declination ascension of the source. Must have astropy angular units
+
+    Returns
+    -------
+    theta : `float/array`
+        inclination from the pole of the source in the spherical
+        projection on the sky. Must have astropy angular units
+
+    phi : `float/array`
+        azimuth of the source in the spherical projection on the sky. Must
+        have astropy angular units
+    """
+    from astropy.coordinates import SkyCoord
```

Comment:
I'd stick the imports at the top of the file. Love the use of SkyCoord though, very slick!!

Reply:
Moved.